src/index.html

```diff
@@ -21,6 +21,7 @@
       href="//fonts.googleapis.com/css?family=Roboto:300,400,700,300italic|Inconsolata:400,700|Roboto+Condensed:400,700"
       type="text/css"
     />
+    <!-- PLACEHOLDER_RUM -->
```

Comment:
analytic scripts are usually placed at the bottom right before the closing `</body>` tag to prevent blocking the rest of the page

Reply:
Google explicitly asks users to put it as the first element in the `<head>`. And I can imagine that it makes sens eto do so in order to be able to track events like first contentful paint etc.

Reply:
So we should blame google making our internet slow! :sweat_smile: